### PR TITLE
15.0 mig account payment term discount

### PR DESCRIPTION
**Pull request by cshardey**

### CODE REVIEW

**Comment by leemannd** [comment]
Hello @cshardey , Thanks for the work. I have made a PR to fix the migration and the test. Is it possible for you to check it and merge it?
https://github.com/cshardey/account-payment/pull/1

**Comment by cshardey** [comment]
Thank you Denis

File: account_payment_term_discount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2010-2012 Camptocamp Austria (<http://www.camptocamp.at>)
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Payment Terms Discount",
+    "version": "15.0.1.1.5",
```
Comment (leemannd) [inline account_payment_term_discount/__manifest__.py:7]:
```suggestion
    "version": "15.0.1.0.0",
```

File: account_payment_term_discount/__manifest__.py [hunk 2]
```diff
@@ -4,10 +4,10 @@
 
 {
     "name": "Account Payment Terms Discount",
-    "version": "14.0.1.1.7",
+    "version": "15.0.1.1.5",
     "license": "AGPL-3",
     "category": "Accounting & Finance",
-    "depends": ["purchase", "account"],
+    "depends": ["purchase", "account", "stock"],
```
Comment (leemannd) [inline account_payment_term_discount/__manifest__.py]: Any reason to add a new stock dependency?

**Comment by cshardey** [comment]
I had an error with the product template view. Apparently, I thought it was
a dependency issue and I realized, it was not.
 I forgot to remove that dependency.

**Review by leemannd — APPROVED** [review]
Base work looks good, but the commits should be cleaned a bit (squashed together)

**Pull request closed without merging**